osrelease/install.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+set -euo pipefail
+
+DIR=~jobrunner/osrelease
+CODE=$DIR/code
+VENV=$DIR/venv
+WHEELS=$DIR/wheels
+CONFIG=$DIR/config.py
+
+mkdir -p $DIR
+test -d $CODE || git clone https://github-proxy.opensafely.org/opensafely-core/output-publisher $CODE
+test -d $WHEELS || git clone https://github-proxy.opensafely.org/opensafely-core/output-publisher-wheels $WHEELS
+test -f $CONFIG || cp osrelease/secrets-template.py $CONFIG
```

Comment:
why `secrets-template.py` and not `config-template.py`, if it's to be copied to `config.py`?

Reply:
(nitpick)